File: backend/internal/core/domain/report.go
package domain

import "time"

type ReportSummary struct {
	ReceiptsCount int `json:"receiptsCount"`
	TotalSum int `json:"totalSum"`
	CashSum int `json:"cashSum"`
	CardSum int `json:"cardSum"`
	IsStub bool `json:"isStub"`
	Note string `json:"note,omitempty"`
}

type Report struct {
	ImportID string
	Status ImportStatus
	Data ReportSummary
	CreatedAt time.Time
	UpdatedAt time.Time
	ErrorCode string
}

File: frontend/shared/types/report.ts
import type { ImportStatus } from './import'

export interface ReportSummary {
  receiptsCount: number
  totalSum: number
  cashSum: number
  cardSum: number
  isStub: boolean
  note?: string
}

export interface Report {
  importId: string
  status: ImportStatus
  data: ReportSummary
  createdAt?: string
  updatedAt?: string
  errorCode?: string
}

export interface IlliquidItem {
  sku: string
  name: string
  category?: string
  stock: number
  daysWithoutSale: number
  lastSaleAt?: string
  recommendation?: 'discount' | 'bundle' | 'writeoff' | 'monitor'
  recommendationNote?: string
}

export interface ReportInsightsResponse {
  importId: string
  generatedAt: string
  items: IlliquidItem[]
}
